features/support/hooks.rb

```diff
@@ -6,14 +6,26 @@
   skip_this_scenario unless ENV['CONJUR_VERSION'] == "4"
 end
 
+Before("@enable-space-host") do
```

Comment:
What happens if you run the cukes without specifying either of these flags? Would you end up w/ a null-ref exception? If so perhaps we should have `@space_host_enabled` default to `false` somewhere and allow this `Before` hook to override it?

Reply:
With cukes, the `~` means that this hook matches if the tag *isn't* applied to a scenario. So this is affectively setting the default to false unless the tag is present.

Reply:
Oh I see, I thought the `~` case would only trigger if you explicitly provided it in the command to run cucumber, but if that's not the case then looks good to me!